doc/xds-test-descriptions.md

```diff
@@ -17,6 +17,11 @@ Server should accept these arguments:
     *   When set to true it uses XdsServerCredentials with the test server for security test cases.
         In case of secure mode, port and maintenance_port should be different.
 
+In addition, when handling requests, if the initial request metadata contains the `rpc-behavior` key, it should modify its handling of the request as follows:
```

Comment:
This is a CSV header, so servers should split on `,` and act as if multiple headers were a single string separated by `,`.

Java processes these immediately, so `sleep-10,fail-on=blah` would sleep for 10 seconds before failing if hostname is blah whereas `fail-on=blah,sleep-10` would fail immediately if hostname is blah. I don't know if other languages do that, but it seems easy to implement and easy to define and reasonably powerful.

Reply:
I didn't explicitly mention the CSV format because the protocol spec already says that separate metadata entries with the same key are semantically equivalent to a comma-separated list of values in a single metadata entry.

Reply:
I did add a note about applying multiple values in order.

Reply:
Sure, yeah. Anything that implies that we support multivalue is good.

Reply:
I don't see anything that says this field is a CSV. And I see now we don't say we support multivalue headers. It is just "multiple values" but in a generic sense. We could say that `rpc-behavior` is a comma-separated list and multiple headers must be supported.

The RFC does say how to combine multiple headers into one, but I don't think it would be clear to implementors of this that we actually expect that to work unless we explicitly mention it. The RFC does not say that every header must support the concatenation semantics. It just says that if there are duplicate keys, they can be combined together without impacting their semantics.